docs/source/NEWS.rst

```diff
@@ -5,6 +5,8 @@ Changelog
 -------------------
 
 - Dropped support for Python 3.5
+- The Deferred returned from send_multiResponse and send_multiResponse_ex is
+  now fired after the final response is handled.
```

Comment:
Open to suggestions on better wording here. I wasn't sure how to say this differently without getting too technical for release notes.